components/camel-sip/src/main/java/org/apache/camel/component/sip/listener/SipPresenceAgentListener.java

```diff
@@ -46,22 +50,26 @@
     protected Dialog dialog;
     protected int notifyCount;
     private SipPresenceAgent sipPresenceAgent;
+    private Queue<String> messageQueue;
```

Comment:
@nikvaessen could you store whole SIP requests in this Queue, rather than just the String body?